### DISCUSSION TITLE
var/cache/prod/pools

### DISCUSSION THREAD

**Discussion by josephtingiris** [kimai/kimai#2795] [Questions & Answers] [answered]
On my install, var/cache/prod/pools is getting rather large.  What's the purpose of this directory?  Is it safe to delete?

**Comment by kevinpapst** [comment, marked as answer]
You can delete it, but it will be recreated on the next request.

**Reply by josephtingiris** [reply]
That worked, thanks.  If anyone else gets there ... removing the pools directory doesn't require a restart either.
